### DISCUSSION TITLE
KusionStack + KCL 18th Weekly Meeting | 第 18 次周会 (2023.07.20)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#3] [General]
# 会议内容
KusionStack + KCL 开源之夏、GLCC 工作内容同步
KusionStack + KCL 近期开发日常工作同步

# 会议方式

Peefy 邀请您参加腾讯会议
会议主题：KusionStack + KCL 社区双周会
会议时间：2023/07/20 20:00-20:45 (GMT+08:00) 中国标准时间 - 北京
重复周期：2023/07/20-2024/08/29 20:00-20:45, 每两周 (周四)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/Hc6sNpqTWnPb

#腾讯会议：778-2381-6338

复制该信息，打开手机腾讯会议即可参与

**Comment by RoyHarringtonnn** [comment]
L'objectif principal de cette réunion est de synchroniser les avancées d'Open Source Summer et de GLCC, et de présenter les développements récents des deux équipes. Elle se déroule en ligne via Tencent Meeting, toutes les deux semaines. Elle s'adresse clairement aux contributeurs et aux membres de la communauté souhaitant rester informés [bdmonline.fr](https://bdmonline.fr/) des projets et des plans en cours. Le format est simple, concis et axé sur le partage d'informations plutôt que sur des discussions techniques approfondies.
